### PR TITLE
perf setup update: add grafana, misc fixes

### PR DESCRIPTION
**Pull request by ldemailly**
need git pull --tags to get latest_release tag movement for fortio on vm, use DUR variable for duration

### CODE REVIEW

File: tools/setup_perf_cluster.sh [hunk 1]
```diff
@@ -134,6 +134,11 @@ function install_istio() {
   Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl apply -f -'
 }
 
+function install_istio_addons() {
```
Comment (Tahler) [inline tools/setup_perf_cluster.sh]: This function isn't called, are you sure you meant to add it?

Comment (ldemailly) [reply tools/setup_perf_cluster.sh]: yes that's the wip part, I will add it to setup all but need to manually test first, if you have time you could get that branch and give it a shot

Comment (ldemailly) [reply tools/setup_perf_cluster.sh]: fixed now

File: install/kubernetes/templates/addons/grafana.yaml.tmpl [hunk 2]
```diff
@@ -4,6 +4,8 @@ kind: Service
 metadata:
   name: grafana
   namespace: {ISTIO_NAMESPACE}
+  annotations:
```
Comment (douglas-reid) [inline install/kubernetes/templates/addons/grafana.yaml.tmpl]: WARNING: this is the wrong place to edit the templates.

you want to edit: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

File: install/kubernetes/helm/istio/charts/grafana/templates/service.yaml [hunk 3]
```diff
@@ -3,6 +3,8 @@ kind: Service
 metadata:
   name: grafana
   namespace: {{ .Release.Namespace }}
+  annotations:
+    auth.istio.io/{{ .Values.service.externalPort }}: NONE
```
Comment (ldemailly) [inline install/kubernetes/helm/istio/charts/grafana/templates/service.yaml]: @douglas-reid : here, PTAL

File: install/kubernetes/templates/addons/grafana.yaml.tmpl (hunk 2 above)
Comment (ldemailly) [reply install/kubernetes/templates/addons/grafana.yaml.tmpl]: yes I know, I commented in #5091 - added the other one above, thanks for the pointer
(while we have 2 I prefer to make both changes to increase the chances that it does work)

Comment (douglas-reid) [reply install/kubernetes/templates/addons/grafana.yaml.tmpl]: OK. This sounds reasonable. We desperately need to untangle this mess.

File: tools/setup_perf_cluster.sh [hunk 4]
```diff
@@ -113,7 +112,7 @@ function delete_vm_firewall() {
 }
 
 function update_fortio_on_vm() {
-  run_on_vm 'go get istio.io/fortio && cd go/src/istio.io/fortio && git fetch && git checkout latest_release && go build -o ~/go/bin/fortio -ldflags "-X istio.io/fortio/version.tag=$(git describe --tag --match v\*) -X istio.io/fortio/version.buildInfo=$(git rev-parse HEAD)" . && sudo setcap 'cap_net_bind_service=+ep' `which fortio` && fortio version'
+  run_on_vm 'go get istio.io/fortio && cd go/src/istio.io/fortio && git fetch --tags && git checkout latest_release && make submodule-sync && go build -o ~/go/bin/fortio -ldflags "-X istio.io/fortio/version.tag=$(git describe --tag --match v\*) -X istio.io/fortio/version.buildInfo=$(git rev-parse HEAD)" . && sudo setcap 'cap_net_bind_service=+ep' `which fortio` && fortio version'
```
Comment (rkpagadala) [inline tools/setup_perf_cluster.sh]: make submodule-sync ??? I thought this is no longer needed.

Comment (ldemailly) [reply tools/setup_perf_cluster.sh]: it is, on fortio

**Pull request merged**